PageSpeed_Api.php

```diff
@@ -1,128 +1,1898 @@
+									then “Coverage”.
+									
+									%2$sA Coverage will open up. We will see buttons for start capturing 
+									coverage, to reload and start capturing coverage and to stop capturing coverage and show 
+									results.
+									
+									%3$sIf you have a webpage you want to analyze its code coverage. Load the webpage 
+									and click on the o button in the Coverage tab.
+									
+									%4$sAfter sometime, a table will show up in 
+									the tab with the resources it analyzed, and how much code is used in the webpage. All the 
+									files linked in the webpage (css, js) will be listed in the Coverage tab. Clicking on any 
+									resource there will open that resource in the Sources panel with a breakdown of Total Bytes 
+									and Unused Bytes.
+									
+									%5$sWith this breakdown, we can see how many unused bytes are in our CSS 
+									files, so we can manually remove them',
+								'w3-total-cache'
+							),
+							'<br/><br/>',
+							'<br/><br/>',
+							'<br/><br/>',
+							'<br/><br/>',
+							'<br/><br/>',
+						),
+						array(
+							'br' => array(),
+						)
+					),
+				),
+				'unminified-css'            => array(
+					'title'        => $this->v( $data, array( 'lighthouseResult', 'audits', 'unminified-css', 'title' ) ),
+					'description'  => $this->v( $data, array( 'lighthouseResult', 'audits', 'unminified-css', 'description' ) ),
+					'score'        => $this->v( $data, array( 'lighthouseResult', 'audits', 'unminified-css', 'score' ) ),
+					'displayValue' => $this->v( $data, array( 'lighthouseResult', 'audits', 'unminified-css', 'displayValue' ) ),
+					'details'      => $this->v( $data, array( 'lighthouseResult', 'audits', 'unminified-css', 'details', 'items' ) ),
+					'type'         => array(
+						'FCP',
+						'LCP',
+					),
+					'instruction'  => wp_kses(
+						sprintf(
+							// translators: 1 opening HTML a tag to W3TC Minify admin page,
+							// translators: 2 closing HTML a tag, 3 opening HTML acronym tag, 4 closing HTML acronym tag,
+							// translators: 5 opening HTML acronym tag, 6 closing HTML acronym tag, 7 opening HTML acronym tag,
+							// translators: 8 closing HTML acronym tag, 9 opening HTML a tag to FAQ page on api.w3-edge.com,
+							// translators: 10 opening HTML acronym tag, 11 closing HTML acronym tag, 12 closing HTML acronym tag.
+							__(
+								'On the %1$sMinify%2$s tab all of the recommended settings are preset. Use the help button 
+									to simplify discovery of your %3$sCSS%4$s and %5$sJS%6$s files and groups. Pay close 
+									attention to the method and location of your %7$sJS%8$s group embeddings. See the 
+									plugin\'s %9$s%10$sFAQ%11$s%12$s for more information on usage.',
+								'w3-total-cache',
+							),
+							'<a href="admin.php?page=w3tc_minify#css" alt="' . __( 'Minify', 'w3-total-cache' ) . '" target="_blank">',
```

Comment:
May need to `esc_attr` the attributes... e.g. a translator could add quotes and mess up the markup.